### DISCUSSION TITLE
Possible temp read bug?

### DISCUSSION THREAD

**Discussion by ha11oga11o** [rern/rAudio#1837] [Fixed Issues]
It seems that last two updates i did reads cpu is overtemp? Is this bug? Its in big aluminium case and theres no way its hot. Underpower it is, that i cant solve for now.

![image](https://github.com/user-attachments/assets/5825bf05-aaff-4cbc-9460-21cffed07a88)

**Comment by rern** [comment]
https://www.raspberrypi.com/documentation/computers/os.html#get_throttled

**Comment by ha11oga11o** [comment]
Im sorry,

i checked before and i cant say i understand. Dont get me wrong. All is working fine. Perfectly. I just wanted to point out that it says CPU temp limit when its obvious its about 40.8C after reboot. Even now, after about 3hrs its sits stable on same temp. 

I am getting code its throttled and thats expected since my psu is not good atm so i used whatever i had on hand. 

Simply dont understand why its pointing "CPU temperature limit" as shown on picture.

Thank you for all your work. :)

# vcgencmd get_throttled
throttled=0x50000

**Comment by markblinkhorn** [comment]
I have a Pi 3 B+ in a fully enclosed case which doesn't get much airflow. The CPU temperature usually hovers around 48C when it is playing. The Pi 3 B+ has a 'soft' temperature limit which is set at 60C by default. The CPU temprature will easily push above this limit if I make it work a bit harder - for example, by doing a Library Update.

Try adding the following line to /boot/config.txt

`temp_soft_limit=65`

You can set the temp anywhere between 60 and the 'hard' limit of 75. Anything above this is overridden by the hard 75 limit.

**Reply by ha11oga11o** [reply]
Thanks for suggestion. For giggles i tried to 75, but have same warning. 

It still works totally fine thou.

**Reply by markblinkhorn** [reply]
Silly question: you did reboot? I ask only because I was having exactly the same issue with my Pi3 B+ and setting the soft limit did resolve it for me. Also, I wasn't having any other problems either, just the annoying message.

**Reply by ha11oga11o** [reply]
Hahaha, of course i reboot. It takes 30 sec to boot, im doing it by default. 

Good one.

**Reply by markblinkhorn** [reply]
I told you it was a silly question ;-)
Just for completeness, I'm running:
`Linux rAudio3 6.12.1-1-rpi #1 SMP PREEMPT Mon Nov 25 13:01:12 MST 2024 aarch64 GNU/Linux`

**Reply by ha11oga11o** [reply]
OK, what am i missing here?

`Linux version 6.1.58-2-rpi-ARCH (builduser@leming) (aarch64-unknown-linux-gnu-gcc (GCC) 12.1.0, GNU ld (GNU Binutils) 2.38) #1 SMP PREEMPT Fri Oct 20 14:01:47 MDT 2023`

I am doing updates from addons/Updates for rAudio

![image](https://github.com/user-attachments/assets/740b519a-97df-4e74-8810-9897f3aefa62)

![image](https://github.com/user-attachments/assets/548df337-08d4-44d8-beda-64627b90ca18)

Looks that im missing at least an kernel update? 

Thanks in advance

**Reply by markblinkhorn** [reply]
You can update the Linux by running:
`pacman -Syu`

Good idea to take a full backup first...

**Reply by ha11oga11o** [reply]
It seems i cant

rAudio:~  #  pacman -Syu
:: Synchronizing package databases...
 +R                                         5.7 KiB  8.69 KiB/s 00:01 [#######################################] 100%
 core                                     228.1 KiB   144 KiB/s 00:02 [#######################################] 100%
 extra                                      9.3 MiB  3.04 MiB/s 00:03 [#######################################] 100%
 alarm                                     57.4 KiB   102 KiB/s 00:01 [#######################################] 100%
:: Starting full system upgrade...
:: Replace dbus-python with extra/python-dbus? [Y/n] y
warning: linux-rpi: ignoring package upgrade (6.1.58-2 => 6.12.4-1)
:: Replace tracker3 with extra/tinysparql? [Y/n] n
resolving dependencies...
looking for conflicting packages...
:: tinysparql and tracker3 are in conflict. Remove tracker3? [y/N] n
error: unresolvable package conflicts detected
error: failed to prepare transaction (conflicting dependencies)
:: tinysparql and tracker3 are in conflict (tracker3<=3.7.3-2)

**Comment by markblinkhorn** [comment]
I've seen that message. IIRC I elected to remove tracker3 - but, then again, I might have done a clean install and then restored the backed up configuration. I have multiple installations of rAudio, so it is even possible that I've done both of these. :D

Good luck

Mark

________________________________

**Reply by ha11oga11o** [reply]
Thanks for tipping in,


but actually im curious about this line:

warning: linux-rpi: ignoring package upgrade (6.1.58-2 => 6.12.4-1)

Cheers.

**Reply by markblinkhorn** [reply]
The default /etc/pacman.conf specifically ignores linux-rpi updates. Edit the conf file to comment out the directive as below:

```

# Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
#IgnorePkg   = linux-rpi
#IgnoreGroup =


```

**Reply by ha11oga11o** [reply]
Thanks,

that help. Still have same issue with temp. It seems its happening all time while temp goes between 40-42C. Something is wrong with Rpi3 probably. Temp reading is wacky probably due hardware issue. Its old and is working 24/7. 

I learned guite a bit thanks to you. Cheers! 

![image](https://github.com/user-attachments/assets/0efda9df-df80-4a32-be0a-f51a2a459a1b)
